pymeasure/instruments/proterial/rod4.py

```diff
@@ -0,0 +1,144 @@
+    )
+    setpoint = Channel.control(
+        "\x020{ch}RFD", "\x020{ch}SFD%.1f",
+        """Control the set point in % of control range.""",
+        validator=truncated_range,
+        values=[0, 100],
+    )
+    mfc_range = Channel.control(
+        "\x020{ch}RFK", "\x020{ch}SFK%d",
+        """Control the MFC range in sccm.
+        Upper limit is 200 slm.""",
+        validator=truncated_range,
+        values=[0, 200000]
+    )
+    ramp_time = Channel.control(
+        "\x020{ch}RRT", "\x020{ch}SRT%.1f",
+        """Control the MFC set point ramping time in seconds.""",
+        validator=truncated_range,
+        values=[0, 200000]
+    )
+    valve_mode = Channel.control(
+        "\x020{ch}RVM", "\x020{ch}SVM%d",
+        """Control the MFC valve mode.
+        Valid options are `flow`, `close`, and `open`. """,
+        validator=strict_discrete_set,
+        values={'flow': 0, 'close': 1, 'open': 2},
+        map_values=True
+    )
+
+    def __init__(self, *argv, **kwargs):
+        super().__init__(*argv, **kwargs)
+        self._flow_unit = None
+
+    @property
+    def flow_unit(self):
+        """Control flow units for the selected channel.
+        Valid options are %, sccm, or slm.
+        Display in absolute units is in sccm for control range < 10 slm."""
+        if self._flow_unit is None:
+            log.info("Flow units have not been set on ROD-4 channel "
+                     f"{self.id}")
+        return self._flow_unit
+
+    @flow_unit.setter
+    def flow_unit(self, units):
+        values = {'%': 0, 'sccm': 1, 'slm': 1}
+        if isinstance(units, str) and units.casefold() in values:
+            units = units.casefold()
+            self.write("\x020{ch}SFU%d" % values[units])
+            self._flow_unit = 'sccm'
+        else:
+            log.info('Units must be %, sccm, or slm.')
+
+
+class ROD4(Instrument):
+    """Represents the Proterial ROD-4(A) operator for mass flow controllers
+    and provides a high-level interface for interacting with the instrument.
+    User must specify which channel to control (1-4).
+
+    .. code-block:: python
+
+        rod4 = ROD4("ASRL1::INSTR")
+
+        print(rod4.version)             # Print version and series number
+        rod4.ch_1.mfc_range = 500       # Sets Channel 1 MFC range to 500 sccm
+        rod4.ch_2.valve_mode = 'flow'   # Sets Channel 2 MFC to flow control
+        rod4.ch_3.setpoint = 50         # Sets Channel 3 MFC to flow at 50% of full range
+        print(rod4.ch_4.actual_flow)    # Prints Channel 4 actual MFC flow in %
+
+    """
+    ch_1 = Instrument.ChannelCreator(ROD4Channel, 1)
+    ch_2 = Instrument.ChannelCreator(ROD4Channel, 2)
+    ch_3 = Instrument.ChannelCreator(ROD4Channel, 3)
+    ch_4 = Instrument.ChannelCreator(ROD4Channel, 4)
+
+    def __init__(self, adapter, name="ROD-4 MFC Controller", **kwargs):
+        super().__init__(
+            adapter, name, read_termination='\r', write_termination='\r', **kwargs
+        )
```

Comment:
It is mandatory to specify, whether it supports SCPI or not:
```suggestion
        super().__init__(
            adapter, name, read_termination='\r', write_termination='\r',
            includeSCPI=False,
            **kwargs)
```